### DISCUSSION TITLE
Buttons not rendering on 3.2.6-canary.1 onward

### DISCUSSION THREAD

**Discussion by streamworksaudio** [muxinc/media-chrome#969] [Q&A]
I think there is a bug that starts with 3.2.6-canary.1 and up.  The bug is that the media-settings-menu-button and media-captions-menu-button buttons are not rendering correctly.

They appear perfect on 3.2.6-canary.0 - but any version after that they do not show on the media-control-bar

Appears to be a bug - anybody else having this issue?

**Comment by luwes** [comment]
This was a change we made for v4. See https://www.media-chrome.org/docs/en/migration/from-3.x-to-4.0

They can be imported via `import 'media-chrome/menu';` 


> Menu components moved to separate import

> The menu components that were included by default in v3 are now moved to a separate import. This gives you more granular control and can save some extra weight in the final JS bundle if you don’t need these components. You can import them from media-chrome/menu.

**Reply by streamworksaudio** [reply]
Thanks for the reply,

I am currently using Media Chrome from the jsdelvir CDN, is there some sort of documentation that shows how I would import these components when using a CDN?

**Reply by rudemateo** [reply]
further down that page, yes: https://www.media-chrome.org/docs/en/migration/from-3.x-to-4.0#deprecated-experimental-selectmenu-related-components

```
<script type="module" src="https://cdn.jsdelivr.net/npm/media-chrome@4/+esm"></script>
<script type="module" src="https://cdn.jsdelivr.net/npm/media-chrome@4/menu/+esm"></script>
```

**Reply by streamworksaudio** [reply]
Perfect!  Thanks for the help
